manifests/helm/templates/deployment.yaml

```
@@ -0,0 +1,125 @@
+    {{- include "go-broker.labels" . | nindent 4 }}  
+spec:  
+  replicas: {{ .Values.replicaCount }}  
+  selector:  
+    matchLabels:  
+      {{- include "go-broker.selectorLabels" . | nindent 6 }}  
+  template:  
+    metadata:  
+      labels:  
+        {{- include "go-broker.selectorLabels" . | nindent 8 }}  
+      annotations:  
+        checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}  
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)